### PR TITLE
New confluence environment

### PR DESCRIPTION
**Pull request by he8us**
[see: ADQ-837]

### CODE REVIEW

**Comment by he8us** [comment]
@rkleiren @laudeco

File: README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ $ docker-compose up -d
 
 ### Use host names instead of ports
 
-If you've launched the environment you already have a proxy running to redirect atlassian-confluence.dev and atlassian-connect.dev to the correct containers.
+If you've launched the environment you already have a proxy running to redirect ngrok.dev and atlassian-connect.dev to the correct containers.
```
Comment (laudeco) [inline README.md]: Why ngrok ?
Does it mean something ?

File: README.md [hunk 2]
```diff
@@ -43,6 +43,22 @@ $ docker-machine ip [machine-name]
...
+
+### Exposing our local app to the world
+
+The Atlassian app we trying to authenticate must post some information to us, so we need to expose our app on the internet. That's the reason we have a [ngrok](https://ngrok.com/) container running. ngrok is an application which will create a tunnel between our environment and their servers, letting us to be accessed from everywhere.
+  
+You should now have a ngrok container running at [ngrok.dev](http://ngrok.dev). When you connect you should see the tunnel url and, if you open it, all the tunnel trafic.
+
+The url to use to install our demo application is https://[your-tunnel-id].eu.ngrok.io/descriptor.json 
+
+__Note:__ Everytime you restart the ngrok container, the tunnel id will change, so you should reinstall your plugin each time. I know, it's bad.
```
Comment (laudeco) [inline README.md]: Where do you save the content of ngrok ? in a data container, if yes why we can not keep it ?

File: docker-compose.yml [hunk 3]
```diff
@@ -50,14 +49,12 @@ services:
...
-            confluence:
-                aliases:
-                    - atlassian-connect.dev
+            - proxy
+            - backend
 
 
     php:
-        image: he8us/php-fpm-prod
+        image: he8us/php-fpm-dev
```
Comment (rkleiren) [inline docker-compose.yml]: don't use your personal repo

**Comment by laudeco** [comment]
ok

File: docker-compose.yml (hunk 3 above)
Comment (he8us) [inline docker-compose.yml]: We already had this discussion, I support my images, and I don't want to support the exact same thing twice. The official PHP-FPM needs some configuration so we need an other one on top with that config.

My image contains pdo_mysql, intl, mbstring, mcrypt, bcmath and xdebug. Which are not in the official image

Comment (laudeco) [inline docker-compose.yml]: And ADF needs those dependencies

**Pull request merged**